src/exchange/ExchangeHomeScreen.tsx

```diff
@@ -171,23 +155,7 @@ function ExchangeHomeScreen() {
           </View>
 
           <CeloGoldHistoryChart />
-          {isCeloNewsEnabled ? (
-            <CeloNewsFeed />
-          ) : (
-            <>
-              {!inAppSwapsEnabled &&
-                (RESTRICTED_CP_DOTO ? (
-                  <RestrictedCeloExchange onPressWithdraw={goToWithdrawCelo} />
-                ) : (
-                  <CeloExchangeButtons />
-                ))}
-              <ItemSeparator />
-              <CeloGoldOverview testID="ExchangeAccountOverview" />
-              <ItemSeparator />
-              <SectionHead text={t('activity')} />
-              <TransactionsList feedType={FeedType.EXCHANGE} />
-            </>
-          )}
+          {isCeloNewsEnabled && <CeloNewsFeed />}
```

Comment:
We can remove `isCeloNewsEnabled` too, right?

Reply:
I debated this, but I though it might be a good idea to have a way to disable the news feed if we ever need it.

Reply:
If for some reason we lost access to our account for social media, it would be great to have a switch for disabling the news feed.